third_party/nvfuser/runtime/basic_type_traits.cu

```diff
@@ -0,0 +1,172 @@
+namespace std {
```

Comment:
Should we put a copyright section and a pointer to the original code here for LLVM copied code?

Reply:
added